.github/workflows/main.yml

```
@@ -22,10 +24,20 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - uses: docker/build-push-action@v1.1.0
+      - name: Set up Docker Buildx
+        id: setup-buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          platforms: |
+            linux/amd64
+            linux/arm64
+
+      - uses: docker/build-push-action@v4
```

il faut également utiliser les meta-données je crois pour faire fonctionner buildx
```suggestion

      - name: Docker meta
        id: meta
        uses: docker/metadata-action@v4
        with:
          images: lephare/apache
          tags: |
            type=raw,${{ matrix.php_version }}


      - uses: docker/build-push-action@v4
```